galley/pkg/config/processing/snapshotter/analyzingdistributor.go

```diff
@@ -156,6 +156,7 @@ func (d *AnalyzingDistributor) analyzeAndDistribute(cancelCh chan struct{}, name
 			msgs = append(msgs, m)
 		}
 	}
+	msgs = msgs.Sorted()
```

Comment:
Thinking about this more, maybe we should just update the for loop to be against ctx.messages.Sorted()? Possibly a tad less efficient to do the sort first, but IMO more idiomatic.

Reply:
If it's on the for loop then we wouldn't sort things without a namespace, but I did move the call to StatusUpdater.Update